model/pdata/common.go

```diff
@@ -401,6 +401,9 @@ func (a AttributeValue) AsString() string {
 		jsonStr, _ := json.Marshal(a.MapVal().AsRaw())
 		return string(jsonStr)
 
+	case AttributeValueTypeBytes:
+		return string(a.BytesVal())
```

Comment:
To be compatible with specs and allow Prometheus to use this we need to base64 this

Reply:
https://github.com/open-telemetry/opentelemetry-specification/issues/780

Reply:
It's unclear from the discussion in that issue whether base64 encoding is the way the specification will move forward. I'm happy to change the implementation to base64 encode the bytes, but want to ensure this will be the right decision moving forward, I could be missing some context about this that you have @bogdandrutu.

Reply:
@tigrannajaryan we need some help here since you added the bytes.

Reply:
`AsString` so far is used for 2 use cases: 1) debug printing, 2)for converting the value to non-OTLP formats.  The spec does not define how non-OTLP formats should deal with `bytes`.

We cannot convert `bytes` to `string` directly. Although it is valid in Go it is unusable for both of the use cases we have so far.

I think base64 is the most reasonable approach. It works for our Collector cases and we can also make it part of the spec here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/non-otlp.md

Reply:
👍 sounds good, i've updated the code and will file an issue/pr in the spec.